tsdef/types.d.ts

```diff
@@ -164,8 +164,17 @@ export interface CreateLocalTrackOptions extends MediaTrackConstraints {
   workaroundWebKitBug1208516?: boolean;
 }
 
+export interface NoiseCancellationOptions {
```

Comment:
We should export this in index.d.ts

Reply:
can you clarify, it is already exported from `index.d.ts`

Reply:
I should be able to do this
```ts
import { NoiseCancellationOptions } from 'twilio-video';

const options: NoiseCancellationOptions = {foo};
```

But in order to do this, we need to export NoiseCancellationOptions from tsdef/index.d.ts. Currently, I don't see it there.

Reply:
got it! I will update, thanks, I do have a test app, that uses these, but it did not initialize these types specifically, so I missed this part.